roles/splunk/tasks/install_apps.yml

```diff
@@ -90,7 +90,7 @@
 
 - name: Ensure correct permissions are set
   file:
-    path: "{{ splunk_home }}/{{ app_dest }}/{{ item.name }}"
+    path: "{{ splunk_home }}/{{ app_dest }}"
```

Comment:
If I did not remove this,Ansible will create a empty directory in `splunk_app_deploy_path` with the app name.
This will should have no impact, since the directory where the apps are should be owned by the user splunk is running as.